src/components/PrizePage/PrizeHeader.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+
+import SvgImg from '../SvgImg';
+import wrench from '../../images/wrench.svg';
+
+const useStyles = makeStyles(theme => ({
+	title: {
+		textAlign: 'center',
+		margin: '60px 10px 30px',
+		fontSize: '2em',
+		fontWeight: 700,
+		fontFamily: theme.typography.fontFamily
+	},
+	wrench: {
+		width: '40px',
+		marginTop: '30px'
+	}
+}));
+
+export default function PrizeHeader() {
+	const classes = useStyles();
+
+	return (
+		<Box display='flex' justifyContent='center' alignItems='center'>
+			<SvgImg src={wrench} className={classes.wrench} height={39} width={62} />
+			<Typography variant='h3' component='h2' className={classes.title}>
+				Prize Tracks
+			</Typography>
+			<SvgImg src={wrench} className={classes.wrench} height={39} width={62}
+				style={{ transform: 'rotate(180deg)' }} />
```

Comment:
The wrenches are positioned weirdly when the screen width is low enough to make "Prize Tracks" wrap to two lines, not sure how to fix.

Also, I feel like the wrenches look out of place in the design of the website... thoughts?

Reply:
<img width="375" alt="Screen Shot 2020-06-23 at 12 47 32 AM" src="https://user-images.githubusercontent.com/33593205/85375507-3c49b880-b4eb-11ea-8874-8f24984586df.png">
Here is Google Chrome pretending to be an iPhone 6. I think this looks pretty good on mobile. My first instinct would be not to worry about it.
I don't think the wrenches look out of place, but I also know 0 about style.